build-on-push/scripts/functions

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright 2016 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Common code used in the bash Jenkins job build phases
+
+log_header(){
+    echo
+    echo "======================== $1 ================================="
+    echo
+}
+
+jenkins_cli(){
+  if [ -n "$BOP_JENKINS_CLI" ]; then
+    $BOP_JENKINS_CLI -noCertificateCheck -s "$JENKINS_URL" "${@}"
```

Comment:
It's (among other ways) authenticated with SSH keys in the users' .ssh directory. The necessity to configure the slave this way is mentioned in the README section "Slave configuration" present in the first commit.